models/edr/dbt_artifacts/dbt_artifacts_state.sql

```diff
@@ -0,0 +1,25 @@
+{{
+  config(
+    materialized = 'view',
+    bind=False
+  )
+}}
+
+{% set artifact_models = [
+  "dbt_models",
+  "dbt_tests",
+  "dbt_sources",
+  "dbt_snapshots",
+  "dbt_metrics",
+  "dbt_exposures",
+  "dbt_seeds",
+] %}
+
+{% for artifact_model in artifact_models %}
+select
+  '{{ artifact_model }}' as artifacts_model,
```

Comment:
Instead of returning all the hashes to the memory - let's leverage the db for doing fast hash calculations. I suggest calculating an md5 on all the hashes here (ordered by the hash itself) instead of returning the list of hashes per artifacts table

Reply:
That was my original implementation.
I changed it because of [this](https://github.com/elementary-data/dbt-data-reliability/pull/241#discussion_r1086511999) and [this](https://github.com/elementary-data/dbt-data-reliability/pull/241#discussion_r1085342913).